test/scylla-gdb/run

```diff
@@ -1,5 +1,12 @@
 #! /bin/bash -eu
 
+if [ "$(name -m)" != x86_64 ]; then
```

Comment:
maybe you meant `uname`, not `name`?
I'm not familiar with a tool called "name"...

Reply:
Fixed. How did that happen?!

Reply:
Simple - even if you `set -e` to has bash exit on error, a failure inside a command interpolation `$(name -m)` does not cause bash to exit. It just interpolates to nothing and prints a "command not found" warning which you probably didn't see, and the equality just never succeeds. I guess you didn't check the patch on x86_64?